src/Toolchain/ToolchainProvider.ts

```diff
@@ -316,27 +316,29 @@ export class ToolchainProvider implements vscode.TreeDataProvider<BaseNode> {
   }
 
   public _run(cfg: string): boolean {
+    const [activeToolchainEnv, activeToolchain] =
+      this.checkAvailableToolchain();
+    if (activeToolchainEnv === undefined || activeToolchain === undefined) {
+      return false;
+    }
+
+    const activeToolchainName = activeToolchain.info.name;
+
     /* istanbul ignore next */
     const notifySuccess = () => {
-      vscode.window.showInformationMessage("Onecc has run successfully.");
+      vscode.window.showInformationMessage(
+        `${activeToolchainName} has run successfully.`
+      );
     };
 
     /* istanbul ignore next */
     const notifyError = () => {
-      this.error("Running onecc has failed.");
+      this.error(`Running ${activeToolchainName} has failed.`);
     };
 
-    const [activeToolchainEnv, activeToolchain] =
-      this.checkAvailableToolchain();
-    if (activeToolchainEnv === undefined || activeToolchain === undefined) {
-      return false;
-    }
-
     Logger.info(
       this.tag,
-      `Run onecc with ${cfg} cfg and ${
-        activeToolchain.info.name
-      }-${activeToolchain.info.version?.str()} toolchain.`
+      `Run tools with ${cfg} cfg and ${activeToolchainName}-${activeToolchain.info.version?.str()} toolchain.`
```

Comment:
```suggestion
      `Running ${cfg} with ${activeToolchainName}-${activeToolchain.info.version?.str()}`
```

How about this? It's too long and getting shortened by '...'

Check out this:
![image](https://github.com/Samsung/ONE-vscode/assets/17171963/0a0c9a9c-ff2e-4cbf-b9c4-f42d35129537)